resources.go

```diff
@@ -1805,7 +1805,7 @@ func Provider() tfbridge.ProviderInfo {
 		},
 		JavaScript: &tfbridge.JavaScriptInfo{
 			Dependencies: map[string]string{
-				"@pulumi/pulumi":    "^0.16.9",
+				"@pulumi/pulumi":    "=0.16.11-dev.1547665405",
```

Comment:
@ellismg is this ok?  i could not figure out any way to get npm to actually use the latest dev version without doing this.  both `^0.16.11` and `dev` did not work.